app/models/place_bid.rb

```diff
@@ -35,65 +32,26 @@ def unsaveable_bid
 
   private
 
-  def auction
-    @auction ||= Auction.find(params[:auction_id])
-  end
-
   def auction_available?
-    Presenter::Auction.new(auction).available?
+    auction.available?
```

Comment:
These seem a lot easier to understand! We don't need to distinguish between an AR auction and a presenter at this level. We just get an object that can respond and tell us whether it is available.